src/status_im/chat/styles/message_input.cljs

```diff
@@ -4,27 +4,26 @@
             [status-im.chat.constants :refer [max-input-height
                                               min-input-height
                                               input-spacing-top
-                                              input-spacing-bottom]]))
+                                              input-spacing-bottom]]
+            [status-im.utils.platform :as p]))
 
 (def input-container
   {:flex-direction :column})
 
-(defn input-view [content-height]
+(defn input-view []
   {:flex-direction   :row
    :align-items      :center
    :justify-content  :center
-   :height           (+ (min (max min-input-height content-height) max-input-height)
-                        input-spacing-top
-                        input-spacing-bottom)
+   :height           56
    :background-color color-white})
 
-(defn message-input-container [content-height]
-  {:height           (min (max min-input-height content-height) max-input-height)
-   :margin-top       input-spacing-top
-   :margin-bottom    input-spacing-bottom
-   :flex             1
-   :flex-direction   "column"
-   :margin-right     0})
+(defn message-input-container []
+  {:height         16.5
+   :margin-top     20
+   :margin-bottom  19.5
+   :flex           1
+   :flex-direction :column
+   :margin-right   0})
```

Comment:
not important or obligatory, but you can replace it with `(def message-input-container { ... })`